### PR TITLE
feat: Zookeeper 3.8.4 as LTS

### PR DESCRIPTION
**Pull request by NickLarsenNZ**
# Description

Add Zookeeper 3.8.4 as LTS.

## Definition of Done Checklist

- Not all of these items are applicable to all PRs, the author should update this template to only leave the boxes in that are relevant
- Please make sure all these things are done and tick the boxes
 
```[tasklist]
# Author
- [ ] Changes are OpenShift compatible
- [ ] Helm chart can be installed and deployed operator works
- [ ] Integration tests passed (for non trivial changes)
```

```[tasklist]
# Reviewer
- [ ] Code contains useful comments
- [ ] Code contains useful logging statements
- [ ] (Integration-)Test cases added
- [ ] Documentation added or updated. Follows the [style guide](https://docs.stackable.tech/home/nightly/contributor/docs-style-guide).
- [ ] Changelog updated
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)
```

```[tasklist]
# Acceptance
- [ ] Feature Tracker has been updated
- [ ] Proper release label has been added
- [ ] [Roadmap](https://github.com/orgs/stackabletech/projects/25/views/1) has been updated
```

### CODE REVIEW

**Review by sbernauer — CHANGES_REQUESTED** [review]
Well that was easy... Ran `ack '3\.8\.3'`

File: docs/modules/zookeeper/partials/supported-versions.adoc [hunk 1]
```diff
@@ -3,4 +3,5 @@
 // Stackable Platform documentation.
 
 - 3.9.1
-- 3.8.3 (LTS)
+- 3.8.3
```
[the remark this answers is not in the document]
Comment (NickLarsenNZ) [reply docs/modules/zookeeper/partials/supported-versions.adoc:6]: TIL: I never knew I needed `ack` :D

**Comment by NickLarsenNZ** [comment]
Just holding off merging this until `docker.stackable.tech/stackable/zookeeper:3.8.4-stackable0.0.0-dev` is biult, then I'll run the test suite.

It is currently failing to build: https://github.com/stackabletech/docker-images/actions/runs/8233448524/job/22512946842

**Comment by NickLarsenNZ** [comment]
Tests are passing now

```
=== NAME  kuttl
    harness.go:405: run tests finished
    harness.go:513: cleaning up
    harness.go:570: removing temp folder: ""
--- PASS: kuttl (857.70s)
    --- PASS: kuttl/harness (0.00s)
        --- PASS: kuttl/harness/znode_zookeeper-latest-3.9.1 (30.64s)
        --- PASS: kuttl/harness/smoke_zookeeper-3.8.3_use-server-tls-false_use-client-auth-tls-false (109.78s)
        --- PASS: kuttl/harness/smoke_zookeeper-3.9.1_use-server-tls-true_use-client-auth-tls-false (103.81s)
        --- PASS: kuttl/harness/cluster-operation_zookeeper-latest-3.9.1 (34.31s)
        --- PASS: kuttl/harness/delete-rolegroup_zookeeper-3.9.1 (83.16s)
        --- PASS: kuttl/harness/delete-rolegroup_zookeeper-3.8.4 (87.13s)
        --- PASS: kuttl/harness/delete-rolegroup_zookeeper-3.8.3 (94.18s)
        --- PASS: kuttl/harness/logging_zookeeper-3.9.1 (81.01s)
        --- PASS: kuttl/harness/logging_zookeeper-3.8.4 (66.87s)
        --- PASS: kuttl/harness/logging_zookeeper-3.8.3 (68.55s)
        --- PASS: kuttl/harness/smoke_zookeeper-3.9.1_use-server-tls-true_use-client-auth-tls-true (99.36s)
        --- PASS: kuttl/harness/smoke_zookeeper-3.8.4_use-server-tls-false_use-client-auth-tls-true (110.46s)
        --- PASS: kuttl/harness/smoke_zookeeper-3.9.1_use-server-tls-false_use-client-auth-tls-true (89.33s)
        --- PASS: kuttl/harness/smoke_zookeeper-3.9.1_use-server-tls-false_use-client-auth-tls-false (100.22s)
        --- PASS: kuttl/harness/smoke_zookeeper-3.8.4_use-server-tls-true_use-client-auth-tls-true (89.52s)
        --- PASS: kuttl/harness/smoke_zookeeper-3.8.4_use-server-tls-true_use-client-auth-tls-false (87.96s)
        --- PASS: kuttl/harness/smoke_zookeeper-3.8.3_use-server-tls-true_use-client-auth-tls-true (89.74s)
        --- PASS: kuttl/harness/smoke_zookeeper-3.8.4_use-server-tls-false_use-client-auth-tls-false (88.12s)
        --- PASS: kuttl/harness/smoke_zookeeper-3.8.3_use-server-tls-true_use-client-auth-tls-false (87.51s)
        --- PASS: kuttl/harness/smoke_zookeeper-3.8.3_use-server-tls-false_use-client-auth-tls-true (89.83s)
PASS
~/dev/github.com/stackabletech/zookeeper-operator
```

**Pull request merged**